### ISSUE TITLE
Implement docusaurus website 

### ISSUE THREAD

**Issue by 0aps** [react-spectre/react-spectre#34]
We need to create a docusaurus based website to display the documentation for the end user.

Regards, 
Ángel

**Comment by 0aps** [comment]
Guys,
   I'll integrate a basic setup for a docusaurus based website with couple of examples. I don't have much experience with it so will do the bootstrapping and we'll improve it gradually.

  @rmariuzzo what do you think? do you have any preferable way to set it up?

Regards
Ángel

**Comment by rmariuzzo** [comment]
I'm checking this right now.

**Comment by rmariuzzo** [comment]
Got this so far, very basic: https://react-spectre.github.io/, repo is located here: https://github.com/react-spectre/react-spectre.github.io

**Comment by 0aps** [comment]
great!
Good thing I didn't start because I wasn't thinking on a different repo. hehe 
But it totally makes more sense that way. 

Only question I have is about duplicating documentation (readme of each package) and the documentation for the end-user. Does it makes sense to implement something which extract/uses the readme of each packages to be displayed in the public page too? In thay way, we don't have to keep up to date both sites docs.

On a side note, maybe it makes sense to move this issue to the other repo. 

Regards, 
Ángel

**Comment by rmariuzzo** [comment]
Indeed, this is a good point. We don't want have to duplicate efforts. Probably, we will move all docs to the public site instead.

**Comment by rmariuzzo** [comment]
I made some progress, but feels like docusaurus will not fit what I want which is be able to easily render our React Components :-(

**Comment by 0aps** [comment]
what do you mean by easily render our React Components? what part aren't we able to do with Docusaurus?

**Comment by rmariuzzo** [comment]
The docs format are rendered in markdown... which I still don't know how to render a React component inside the markdown.

**Comment by rmariuzzo** [comment]
@0aps I'm looking at Gatsby right now and so far it looks promising.
